### DISCUSSION TITLE
[@types/html2json] : Node.attrs instead of Node.attr

### DISCUSSION THREAD

**Discussion by asriayoub** [DefinitelyTyped/DefinitelyTyped#60950] [Issues with a @types package] [answered]
Hello,

I am using this beautiful library, but i found that the parsing generates `Node.attr` instead of `Node.attrs` (as provided by the Typing).

So i'am using an extension for now:
```
interface HTMLNode extends Node {
  child: HTMLNode[] | HTMLNode;
  attr: { [key: string]: string };
}
```

So i think it might be a blocking issue for a bit more flexible use.

Thanks for helping.

[a comment by an automated account is not shown]

**Reply by asriayoub** [reply]
Thanks bot!

**Comment by YEK-PLUS** [comment]
Hi, thanks for your attention. I made new pr with your feedback, thanks.

**Comment by YEK-PLUS** [comment, marked as answer]
@asriayoub it's must be a solved, check out

**Reply by asriayoub** [reply]
Hello sir, it works! Thanks for your quick response! =)
